data_structures/circular_queue_using_binaric_file.cpp

```diff
@@ -0,0 +1,159 @@
+/**
+ * @file
+ * @brief Implementation of a queue using a file.
+ * @details
+ * This program implements a queue using a file.
+ * It includes a template class `MyQueue` defining the queue interface,
+ * and a class `QueueFile` implementing the queue using a file as storage.
+ * * @algorithm
+ * The qoeoe include two pointers to the head and the tail of the qoueue
+ * And it also contain fstraem file there the data of the queue kept.
+ * When removing the value of the head indexOfFirst increase by one
+ * @author [Your Name](https://github.com/your_handle)
```

Comment:
```suggestion
 * @author [chavi](https://github.com/chavi362)
```
you should add your handle to the algorithm